_ont/ont-leox-lxt-010s-h.md

```diff
@@ -12,27 +12,65 @@ parent: Leox
 | Vendor      | Leox                                 |
 | Model       | LXT-010S-H                           |
 | Chipset     | RTL9601CI                            |
-| Flash       |                                      |
-| RAM         |                                      |
+| Flash       | 128MB                                |
+| RAM         | 32MB                                 |
 | System      | Linux 3.18 (Luna SDK 3.3)             |
 | HSGMII      | ✅                                   |
-| Optics      |                                      |
+| Optics      | SC/UPC                               |
 | IP address  | 192.168.100.1/24                     |
-| Web Gui     |                                      |
+| Web Gui     | ✅                                   |
 | Telnet      | ✅ user `leox`, password `leolabs_7` |
 | Form Factor | miniONT SFP                          |
 
 
 ## List of software versions
-## List of partitions
-## List of firmwares and files
+# List of partitions
+
+| dev   | size     | erasesize | name            |
+| ----- | -------- | --------- | --------------- |
+| mtd0  | 00040000 | 00001000  | "boot"          |
+| mtd1  | 00002000 | 00001000  | "env"           |
+| mtd2  | 00002000 | 00001000  | "env2"          |
+| mtd3  | 0003c000 | 00001000  | "config"        |
+| mtd4  | 00300000 | 00001000  | "k0"            |
+| mtd5  | 004c0000 | 00001000  | "r0"            |
+| mtd6  | 00300000 | 00001000  | "k1"            |
+| mtd7  | 004c0000 | 00001000  | "r1"            |
+| mtd8  | 00001000 | 00001000  | "Partition_008" |
+| mtd9  | 00001000 | 00001000  | "Partition_009" |
+| mtd10 | 00001000 | 00001000  | "Partition_010" |
+| mtd11 | 00001000 | 00001000  | "Partition_011" |
+| mtd12 | 00300000 | 00001000  | "linux"         |
+| mtd13 | 004c0000 | 00001000  | "rootfs"        |
+
+This stick supports dual boot. 
+
+`k0` and `r0` respectively contain the kernel and firmware of the first image, `k1` and `r1` the kernel and the firmware of the second one
+
+## Serial Console
+
+The stick has a TTL 3.3v UART console (115200bps - 8Bit) that can be accessed from the top surface. To accept TX line commands, GND of TTL adapter should be attached to the stick's shield:
```

Comment:
I do not remember whether this is the syntax already used on other pages or whether it is different. I don't think there is a capital B of bit on other pages.